__init__.py

```diff
@@ -56,9 +64,12 @@ def handle_intent(self, message):
         except Exception as e:
             LOGGER.error("Error: {0}".format(e))
 
+    def handle_stop(self, message):
+        self.stop()
+        self.speak_dialog('npr.news.stop')
```

Comment:
I think this should be moved back to the stop method so it only speaks "Stopping news" if news was is actually playing.